Makefile

```diff
@@ -54,9 +54,9 @@ else
 # If we are not in build container, we need a workaround to get environment properly set
 # Write to file, then include
 $(shell mkdir -p out)
-$(shell $(PWD)/common/scripts/setup_env.sh envfile > out/.env)
+$(shell $(shell pwd)/common/scripts/setup_env.sh envfile > out/.env)
```

Comment:
I know this is just a test, but assuming this passes and you port the change to `istio/common-files`, should the other `${PWD}` in [`setup_env.sh`](https://github.com/istio/common-files/blob/c46d62d062139196871640f15f04a57b993ef841/files/common/scripts/setup_env.sh#L69-L70) also be changed to `$(pwd)` since you mentioned this is not always set in CI: https://github.com/istio/common-files/pull/208?

```diff
- export TARGET_OUT="${TARGET_OUT:-${PWD}/out/${TARGET_OS}_${TARGET_ARCH}}"
- export TARGET_OUT_LINUX="${TARGET_OUT_LINUX:-${PWD}/out/linux_amd64}"
+ export TARGET_OUT="${TARGET_OUT:-$(pwd)/out/${TARGET_OS}_${TARGET_ARCH}}"
+ export TARGET_OUT_LINUX="${TARGET_OUT_LINUX:-$(pwd)/out/linux_amd64}"
```

Reply:
https://github.com/istio/common-files/pull/209